site/content@beta/platform-overview-new/about-livechat.md

```diff
@@ -0,0 +1,35 @@
+---
+weight: 30
+---
+
+# What LiveChat is
+
+**LiveChat** helps you handle conversations with the customers who visit your websites. Customers can start chats with your agents, but it can also be the other way round. Your agents can initiate chats, making your visitors feel welcome and valued. If you're up to some automation, you can engage them in a conversation with [**greetings**](https://www.livechatinc.com/help/why-should-i-set-up-chat-greetings/). Also, delegating some work to [**bots**](https://www.livechatinc.com/help/chatbots-explained/) is an options when running a bussines on a bigger scale. 
+
+Your customers can start multiple conversations with agents, as well as continue the existing ones - it's up to them. Keeping track of chat history is easy, as agents can scroll up to get a preview of all the exchanged messages in a conversation. 
+
+All of these functionalities are guaranteed *out of the box*, but this is just a tiny bit of what LiveChat offers. If you're interested in other default features, you can check them out[**here**](https://www.livechatinc.com/features/). 
+
+You can also have a quick glance at the list below. It summarizes, from a more tech perspective, the features provided by our product:
+
+|    | Feature |
+|----|--------|
+| ✅|text messaging APIs|
+| ✅|rich messaging APIs|
+| ✅|chat routing APIs|
+| ✅|asynchronous transport (Web API)|
```

Comment:
nie jestem pewny czy w ten sposób można powiedzieć o requestach XHR, wg mnie nie jest to asynchroniczny transport